pandas/util/testing.py

```diff
@@ -2608,6 +2610,9 @@ class for all warnings. To check that no warning is returned,
         If True, displays the line that called the function containing
         the warning to show were the function is called. Otherwise, the
         line that implements the function is displayed.
+    message : str, default ''
```

Comment:
how would one use message?

Reply:
It ends up being hard to use effectively, because unhandled warnings are cause us to raise immediately. Just removing for now, and we can re-add later if needed.